shell/platform/linux/BUILD.gn

```diff
@@ -265,10 +265,11 @@ copy("publish_headers_linux") {
 
 zip_bundle("flutter_gtk") {
   prefix = ""
-  if (flutter_runtime_mode != "debug") {
-    prefix = "$full_platform_name-$flutter_runtime_mode/"
+  if (flutter_runtime_mode != "debug" ||
+      (flutter_runtime_mode == "debug" && target_cpu != "x64")) {
+    prefix = "$platform_name-$target_cpu-$flutter_runtime_mode/"
   }
-  output = "$prefix$full_platform_name-flutter-gtk.zip"
+  output = "${prefix}${platform_name}-${target_cpu}-flutter-gtk.zip"
```

Comment:
The platform name here should be the target os and not the host os because it does not make sense to mix the host os with the target cpu.

Reply:
updated!